tests/integration/api/test_user.py

```diff
@@ -11,21 +11,71 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-from unittest import mock
-
 import pytest
+from ansible_base.rbac.models import RoleDefinition
+from django.contrib.contenttypes.models import ContentType
 from rest_framework import status
+from rest_framework.reverse import reverse
 from rest_framework.test import APIClient
 
 from aap_eda.core import models
-from aap_eda.core.enums import Action, ResourceType
 from tests.integration.constants import api_url_v1
 
 from .conftest import ADMIN_USERNAME
 
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 
 
+@pytest.fixture
```

Comment:
I think we already have fixtures that can be reused in tests/integration/api/conftest.py

Reply:
I went through the list and consolidated what fixtures I could.